RockPaperScissors/RockPaperScissors/RockPaperScissors.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  RockPaperScissors.swift
+//  RockPaperScissors
+//
+//  Created by 박종화 on 2023/05/03.
+//
+
+struct RockPaperScissors {
+    var isFirstGame: Bool = true
+    var isSecondGame: Bool = false
+    var turnDictionary: [Bool: String] = [true: "사용자", false: "컴퓨터"]
+    var whoseTurn: String?
+    
+    mutating func start() {
+        while isFirstGame {
+            do {
+                try startFirstGame()
+            } catch UserCardsError.firstGameInputError {
+                print("잘못된 입력입니다. 다시 시도해주세요.")
+            } catch {
+                print("에러")
```

Comment:
보통 마지막 catch 문에는 error 상수를 사용할 수 있습니다.
print를 통해 특정 문구를 출력하고 싶으시다면 "알 수 없는 오류가 발생했습니다." 가 좋아 보입니다. ☺️

```suggestion
                print(error)
```

Reply:
임시로 "에러"라고만 넣었는데 저희도 세세한 에러 메서지를 넣는 게 더 좋다고 생각합니다. 해당 내용도 수정했습니다!